### PR TITLE
fix(binanceus) - ws extend correct inheritance

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

**Comment by ttodua** [comment]
@carlosmiei PR is ready for your review. 
our transpiler is some bit complex and messy (until we move to ASTT) and at this moment, this seemed less intrusive for me.

File: build/transpileWS.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ class CCXTProTranspiler extends Transpiler {
         return 'class ' + className + '(' +  baseClasses.join (', ') + '):'
     }
 
-    createPythonClassImports (baseClass, async = false) {
+    createPythonClassImports (baseClass, className, async = false) {
```
Comment (ttodua) [inline build/transpileWS.js:40]: this was mistake, because `transpiler.js`s method signature includes className

**Comment by ttodua** [comment]
I've changed all classes that had such chained dependencies , except `binanceusdm`, because it's too much used in ccxt and i am afraid even minor change shouldn't break some userland codes, so just left `binanceusdm` without adding this change.

**Comment by carlosmiei** [comment]
@ttodua what if we added to the base class a method called `createRestInstance` or something? to avoid all the imports and stuff like that

**Pull request merged**